support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -230,12 +230,47 @@ const isCardUserSelected = (
 		return false;
 	}
 	return (
-		(contributionType in paymentFrequencyMap &&
-			Number(urlSelectedAmount) === cardPrice) ||
+		Number(urlSelectedAmount) === cardPrice ||
 		Number(urlSelectedAmount) === cardPriceDiscount
 	);
 };
 
+const productCatalog = window.guardian.productCatalog;
+function getCardData(
+	// TODO - this type could use some refinement
+	productDescription: (typeof productCatalogDescription)[keyof typeof productCatalogDescription],
+	pricing: number,
+	link: string,
+	promotion?: Promotion,
+	isRecommended = false,
+) {
+	return {
+		title: productDescription.label,
+		isRecommended,
+		isUserSelected: isCardUserSelected(pricing, promotion?.discount?.amount),
+		benefits: {
+			list: 'benefits' in productDescription ? productDescription.benefits : [],
+			description: undefined,
+		},
+		planCost: {
+			price: pricing,
+			promoCode: promotion?.name,
+			discount:
+				promotion?.discount?.amount && promotion.discountedPrice
+					? {
+							percentage: promotion.discount.amount,
+							price: promotion.discountedPrice,
+							duration: {
+								value: promotion.numberOfDiscountedPeriods ?? 0,
+								period: 'MONTHLY' as RegularContributionType,
+							},
+					  }
+					: undefined,
+		},
+		externalBtnLink: link,
+	};
+}
```

Comment:
The plan will be to remove this completely and have the `TierCards` render the product directly.